commit 469eab456f98a1815fe7559dc91d32c2dc61f9e2
Author: Miklos Szeredi <mszeredi@redhat.com>
Date:   Wed Feb 22 20:08:25 2017 +0100

    fuse: add missing FR_FORCE
    
    commit 2e38bea99a80eab408adee27f873a188d57b76cb upstream.
    
    fuse_file_put() was missing the "force" flag for the RELEASE request when
    sending synchronously (fuseblk).
    
    If this flag is not set, then a sync request may be interrupted before it
    is dequeued by the userspace filesystem.  In this case the OPEN won't be
    balanced with a RELEASE.
    
    Signed-off-by: Miklos Szeredi <mszeredi@redhat.com>
    Fixes: 5a18ec176c93 ("fuse: fix hang of single threaded fuseblk filesystem")
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1296_469eab456f98a1815fe7559dc91d32c2dc61f9e2.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1296_469eab456f98a1815fe7559dc91d32c2dc61f9e2.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
